styles/styles.css

```diff
@@ -129,6 +129,7 @@
 
   /* Column Split Ratios */
   --split-endorsement: 76%:24%;
+  --split-about-us: 25%:75%;
```

Comment:
@bosschaert what naming convention are we going for with the splits?

Reply:
Also, should we remove the `center content` block now? I think we should

Reply:
> @bosschaert what naming convention are we going for with the splits?

It's a good question. Maybe leave it the way you have it and then if we need the same split elsewhere come up with a common name?


> Also, should we remove the `center content` block now? I think we should

Yes I agree